webpack/JobInvocationDetail/JobInvocationSelectors.js

```diff
@@ -0,0 +1,4 @@
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+
+export const selectItems = state =>
+  selectAPIResponse(state, 'JOB_INVOCATION_KEY') || [];
```

Comment:
since items is used as an object `{ description, status_label: statusLabel, task } = items;`, the fallback should be an empty object `{}`

Reply:
Actually, `selectAPIResponse` already gives a fallback, so no fallback needed.
`export const selectAPIResponse = (state, key) =>
  selectAPIByKey(state, key).response || {};`